packages/components/src/internal/components/assay/SpecialtyAssayPanel.tsx

```diff
@@ -61,15 +72,44 @@ export const SpecialtyAssayPanel: FC<SpecialtyAssayPanelProps> = memo(props => {
                     )}
                 </Col>
             </Row>
-            <Row>
-                <Col xs={6}>
-                    <div className={warning ? 'margin-bottom' : 'margin-top margin-bottom'}>
-                        <b>Supported File Types</b>
-                    </div>
-                    <p>{selected?.fileTypes.join(', ')}</p>
-                </Col>
-            </Row>
+            {selected && options?.length > 0 &&
+                <Row>
+                    <Col xs={6}>
+                        <div className={warning ? 'margin-bottom' : 'margin-top margin-bottom'}>
+                            <b>Supported File Types</b>
+                        </div>
+                        <p>{selected?.fileTypes.join(', ')}</p>
+                    </Col>
+                </Row>
+            }
             {children}
+            {!hasPremium && (
+                <Row>
+                    <Col xs={12} className={noPremiumClass}>
+                        <Alert bsStyle="info">
```

Comment:
Do we have other cases of this Premium Feature upsell alert in other React components? Seems like sometime soon we'll want to componentize this so they they all share the same structure and just pass in props for the body or text/links.

Reply:
The only other use case I know of is in a JSP for EMPI.  I will note it in the release notes and try to watch out for other commits.